src/fsharp/NameResolution.fs

```diff
@@ -2229,44 +2227,30 @@ let rec ResolveLongIdentInTypePrim (ncenv: NameResolver) nenv lookupKind (resInf
     match nestedSearchAccessible with
     | Result res when not (isNil res) -> nestedSearchAccessible
     | _ ->
-        let suggestMembers() =
```

Comment:
This whole bit below is better than constantly creating lists 6 times :)

Reply:
;-)